protocol/x/prices/keeper/msg_server_create_oracle_market_test.go

```diff
@@ -105,6 +114,17 @@ func TestCreateOracleMarket(t *testing.T) {
 			gotAllMarketParamPrices, err := pricesKeeper.GetAllMarketParamPrices(ctx)
 			require.NoError(t, err)
 			require.Equal(t, tc.expectedMarkets, gotAllMarketParamPrices)
+
+			// Check if the market is enabled in MarketMap
+			if len(tc.expectedMarkets) > 0 {
+				for i := range tc.expectedMarkets {
+					market := tc.expectedMarkets[i]
+
+					currencyPair, _ := slinky.MarketPairToCurrencyPair(market.Param.Pair)
+					mmMarket, _ := pricesKeeper.MarketMapKeeper.GetMarket(ctx, currencyPair.String())
+					require.True(t, mmMarket.Ticker.Enabled)
+				}
+			}
```

Comment:
nit: Add same test before the creation that the markets are not enabled?

Reply:
done!